content/en/core/guides/docker-setup.md

```diff
@@ -43,7 +43,7 @@ version: '3.7'
 services:
   medic-os:
     container_name: medic-os
-    image: medicmobile/medic-os:cht-3.7.0-rc.1
+    image: medicmobile/ medic-os:cht-3.9.0-rc.2
```

Comment:
I think you added an extra space here that would break.

```suggestion
    image: medicmobile/medic-os:cht-3.9.0-rc.2
```

Also, maybe we should just say use the latest so it never gets out of date again, eg:

```suggestion
    image: medicmobile/medic-os:latest
```